homeassistant/components/blink/alarm_control_panel.py

```diff
@@ -60,37 +65,24 @@ def __init__(self, data, name: str, sync) -> None:
             manufacturer=DEFAULT_BRAND,
         )
 
-    async def async_update(self) -> None:
-        """Update the state of the device."""
-        if self.data.check_if_ok_to_update():
-            _LOGGER.debug(
-                "Initiating a blink.refresh() from BlinkSyncModule('%s') (%s)",
-                self._name,
-                self.data,
-            )
-            try:
-                await self.data.refresh(force=True)
-                self._attr_available = True
-            except asyncio.TimeoutError:
-                self._attr_available = False
-
-            _LOGGER.info("Updating State of Blink Alarm Control Panel '%s'", self._name)
-
-        self.sync.attributes["network_info"] = self.data.networks
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle coordinator update."""
+        self.sync.attributes["network_info"] = self.api.networks
         self.sync.attributes["associated_cameras"] = list(self.sync.cameras)
         self.sync.attributes[ATTR_ATTRIBUTION] = DEFAULT_ATTRIBUTION
         self._attr_extra_state_attributes = self.sync.attributes
-
-    @property
-    def state(self) -> StateType:
-        """Return state of alarm."""
-        return STATE_ALARM_ARMED_AWAY if self.sync.arm else STATE_ALARM_DISARMED
+        self._attr_state = (
+            STATE_ALARM_ARMED_AWAY if self.sync.arm else STATE_ALARM_DISARMED
+        )
+        self._attr_available = True
+        self.async_write_ha_state()
```

Comment:
```suggestion
        super()._handle_coordinator_update()
```
Not sure if this makes sense, but just throwing it out there

Reply:
What will this do for us?  Call the `_handle_coordinator_update()' for the CoordinatorEntity which is essentially `async_write_ha_state()`.
https://github.com/home-assistant/core/blob/f8ed051f01295cfe288219d88328f9250d6cb9d9/homeassistant/helpers/update_coordinator.py#L466-L469
Which is the "right" way?!?

Reply:
Honestly, not sure, but I have seen this in reviews sometimes as a good practice I think